maps/cynosure/cynosure-2.dmm

```diff
@@ -23426,7 +23423,7 @@
 	},
 /obj/machinery/computer/security/telescreen{
 	name = "Test Chamber Monitor";
-	network = list("Miscellaneous Reseach");
+	network = list("Miscellaneous  Reseach");
```

Comment:
?

Reply:
Honestly no idea why that happened. Dreammaker.